docker/multi-process/Dockerfile

```diff
@@ -1,32 +1,24 @@
 FROM ubuntu:14.04
 
-ADD docker/scripts/prepare /scripts/prepare
+COPY docker/scripts/prepare /scripts/
```

Comment:
Is there a benefit of using `COPY` over `ADD`?

Reply:
The [Best practice guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy) recommends COPY over ADD for the current use case, although I don't know if at the moment there are actual differences between the functionalities of the two directives.

Reply:
Thanks for the clarification.